src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,84 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K, V> [] table;
+    private int size;
+
+    public MyHashMap() {
+        this.size = 0;
+        this.table = new Node[INITIAL_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        int index = getIndex(key);
+        for (Node<K, V> i = table[index]; i != null; i = i.next) {
```

Comment:
Please, give more informative name
```suggestion
        for (Node<K, V> currentNode = table[index]; currentNode != null; currentNode = currentNode.next) {
```